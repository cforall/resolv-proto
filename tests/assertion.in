1 f 1
T g T | T f T

h T | T &x
1 &x
%%
g( 1 )
g( 2 )
h( 1 )
