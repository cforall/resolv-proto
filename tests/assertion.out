
[1 / (0,1,0)]{T<2> => 1 | T<2> f T<2> => f( 1 )} g{T<2> | T<2> f T<2>}( 1 )

[1 / (1,1,0)]{T<1> => 1 | T<1> f T<1> => f( 1 )} g{T<1> | T<1> f T<1>}( 2 => 1 )
